src/imgtools/cli/autopipeline.py

```
@@ -0,0 +1,245 @@
+    if roi_match_map:
+        for mapping in roi_match_map:
+            parts = mapping.split(":", 1)
+            if len(parts) != 2:
+                raise click.BadParameter(f"Invalid ROI match map format: {mapping}")
+            key, patterns = parts
+            roi_map[key] = patterns.split(",")
+
+    from imgtools.autopipeline import DeltaPipeline
+    from imgtools.io.sample_output import ExistingFileMode
+    # Create the pipeline
+    pipeline = DeltaPipeline(
```

Ruff (0.8.2)

Local variable `pipeline` is assigned to but never used

Remove assignment to unused variable `pipeline`

(F841)